### ISSUE TITLE
Selecting text with the mouse has strange behavior

### ISSUE THREAD

**Issue by fibbo** [vscode-neovim/vscode-neovim#891]
With 0.0.84, there is now bizarre behavior when selecting text with the mouse.

![neovim_selection](https://user-images.githubusercontent.com/1519902/166411663-4cf6e85a-0471-466f-8ef3-da7bffb4e979.gif)

All I'm doing is selecting text - no keys are touched. 

There are no problems after reverting to 0.0.83.

Additional info:

* No other extensions are enabled.

* VSCode
```
Version: 1.66.2 (Universal)
Commit: dfd34e8260c270da74b5c2d86d61aee4b6d56977
Date: 2022-04-11T07:49:20.994Z
Electron: 17.2.0
Chromium: 98.0.4758.109
Node.js: 16.13.0
V8: 9.8.177.11-electron.0
OS: Darwin arm64 21.4.0
```
* NVIM:

```
NVIM v0.8.0-dev+112-g13520aae1
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by philippg@mbp.zhinst.com

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/share/nvim"

Run :checkhealth for more info
```

**Comment by jackblackevo** [comment]
I have a same problem.

**Comment by fibbo** [comment]
The commit breaking the text selection: https://github.com/vscode-neovim/vscode-neovim/commit/94b26cc64ee5a9fbef8df55815955a133e6b8cb6 @cspotcode do you have any idea why this is?

**Comment by theol0403** [comment]
All that did was make `vscode-neovim.mouseSelectionStartVisualMode` actually do something - if it is causing problems for you, disable it in your vscode settings.

**Comment by jonpdw** [comment]
Yep

> All that did was make `vscode-neovim.mouseSelectionStartVisualMode` actually do something - if it is causing problems for you, disable it in your vscode settings.

Yep, disabling the setting fixed the issue for me

**Comment by theol0403** [comment]
Please check if this is fixed by #870
